### ISSUE TITLE
Accessibility: Properly handle language selection

### ISSUE THREAD

**Issue by MatzeKitt** [jhaals/yopass#3472]
When navigating to the language "button" as is via keyboard navigation, it directly opens the language selection list. This should only occur on click.

The button should also contain a `aria-expanded`, which should be `true`, if the language selection list is visible. It should also have an `aria-controls` to connect it to the list itself.

See also: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-expanded

**Comment by MatzeKitt** [comment]
Not stale.
